ash/src/device.rs

```diff
@@ -6,7 +6,19 @@ use std::mem;
 use std::os::raw::c_void;
 use std::ptr;
 
-/// <https://www.khronos.org/registry/vulkan/specs/1.3-extensions/man/html/VkDevice.html>
+/// This is a dispatchable object, whose main purpose is to provide access to Vulkan's functions.
+///
+/// It is not identical to Vulkan's device object (see [vk::Device]).
+/// The latter, however, can be accessed via the [`handle()`][Self::handle()] method.
+/// In this sense, `ash::Device` is a Vulkan device handle together with all its associated function pointers.
```

Comment:
Instead of saying what it's not, how about saying what it is? (and don't forget backicks inside `[]` links).

```suggestion
/// Holds a Vulkan [`vk::Device`] handle together with Vulkan 1.0 - 1.3 device function pointers loaded from it, providing convenient wrappers to call these functions.
///
/// The internal handle is available through [`handle()`][Self::handle()] if needed externally.
```